_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -50,26 +53,44 @@ is out of sync with the deployed release.
 
 ### Change deployment properties
 1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
-   that you need to copy over to your previous `deploy-forseti.yaml`. You can use
-   `git diff` to compare what changed. For example, to see the diff between the latest 
-   (HEAD) and one revision ago:
+   that you need to copy over to your previously generated 
+   `deploy-forseti-<timestamp>.yaml`. You can use `git diff` to compare what 
+   changed. For example, to see the diff between the latest (HEAD) and one revision ago:
 
    ```bash
-   $ git diff origin..HEAD~1 -- deploy-forseti.yaml.sample
+   $ git diff origin/master..HEAD~1 -- deploy-forseti.yaml.sample
    ```
 
-1. Edit `deploy-forseti.yaml` and update the values you want to change. If you 
-   previously deployed the `master` branch, you don't need to change it.
+1. Edit `deploy-forseti-<timestamp>.yaml` and update the values of the new properties.
+
+For example, from v1.1.7 to v1.1.10, the following Compute Engine instance 
+properties have changed.
+
+   ```yaml
+   region: $(ref.cloudsql-instance.region)
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+So, you would have to copy these new properties over to your generated
```

Comment:
So, you would have to copy these new properties over to your generated
+`deploy-forseti-<timestamp>.yaml` and update the placeholders to the actual 
+values that you will use, e.g. the project id of the project that Forseti is 
+running in, "default", "default".

To upgrade, copy these new properties to your generated `deploy-forseti-<timestamp>.yaml`. Then, update the placeholders to the values you want to use. For example, the project id of the project that Forseti is running in, "default", "default".

Reply:
Done!